caraml-store-serving/src/main/java/dev/caraml/serving/store/bigtable/BigTableStoreConfig.java

```diff
@@ -23,9 +26,21 @@ public class BigTableStoreConfig {
   private String appProfileId;
   private Boolean enableClientSideMetrics;
   private Long timeoutMs;
+  private Boolean isUsingHBaseSDK;
 
   @Bean
   public OnlineRetriever getRetriever() {
+    // Using HBase SDK
+    if (isUsingHBaseSDK) {
+      org.apache.hadoop.conf.Configuration config =
```

Comment:
should this block wrapped by `try catch` block? I saw that the connect method can throw the IllegalStateException

Reply:
@bayu-aditya could you take a look?

Reply:
Yup sure

Reply:
Done in here https://github.com/caraml-dev/caraml-store/pull/127/commits/455e253feba123d61b371c96206d766192e0acb1 Thank you for your finding

Reply:
should we use one try catch and condition is in the try catch block?